pyproject.toml

```diff
@@ -60,6 +60,7 @@ openai = [ "openai~=1.50.2", "websockets~=13.1", "python-deepcompare~=1.0.1" ]
 openpipe = [ "openpipe~=4.24.0" ]
 playht = [ "pyht~=0.1.4", "websockets~=13.1" ]
 silero = [ "onnxruntime~=1.19.2" ]
+tavus = [ "pipecat[daily,openai,deepgram,cartesia,silero,examples]" ]
```

Comment:
We don't need this. This is only required if `tavus` needed some extra dependencies.